src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/TraceCollection/TraceCollector.cs

```diff
@@ -40,7 +39,14 @@ public sealed class TraceCollector : IDisposable
         {
             { CollectType.gc, "gcCollectOnly" },
             { CollectType.cpu,  "collect_cpu" },
-            { CollectType.threadtime, "collect_threadTime" },
+            { CollectType.threadtime, "collect_threadTime" }
+        };
+
+        internal static readonly Dictionary<CollectType, string> LinuxLocalRunCollectTypeMap = new()
```

Comment:
We should probably rename: https://github.com/dotnet/performance/blob/ab9190fdf173c721c16ef0e56e05dbdd43f71a94/src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure.Core/TraceCollection/TraceCollector.cs#L38 to `LinuxServerRunCollectTypeMap`.